.vimrc

```diff
@@ -129,6 +129,9 @@ endif
 " Vim settings and mappings
 " You can edit them as you wish
 
+" relative numbers
+set relativenumbers
```

Comment:
I get an error message (Vim 7.4.1829 on Ubuntu 16.10):

```
Error detected while processing /home/max/.vimrc:
line  133:
E518: Unknown option: relativenumbers
Press ENTER or type command to continue
```

It does work with `set relativenumber` (no `s`).